Octokit.Reactive/Helpers/ConnectionExtensions.cs

```diff
@@ -28,6 +28,15 @@ public static IObservable<T> GetAndFlattenAllPages<T>(this IConnection connectio
             return GetPages(url, parameters, (pageUrl, pageParams) => connection.Get<List<T>>(pageUrl, pageParams, null).ToObservable());
         }
 
+        public static IObservable<T> GetAndFlattenAllPages<T>(this IConnection connection, Uri url, IDictionary<string, string> parameters, ApiOptions options)
```

Comment:
This all seems fine, and the integration tests are working fine on my side :shipit: